style.css

```diff
@@ -1,648 +1,1323 @@
-  justify-content: left;
-  padding: 10px 12px;
-  background: rgba(255, 255, 255, 0.24);
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 15px;
-  line-height: 20px;
-  height: 40px;
-  border-style: none;
-  color: white;
-}
+  .button1:hover {
+    color: black;
+    background: #fff5e1;
+    border: 1px solid #f55800;
+  }
 
-.button3:hover {
-  box-sizing: border-box;
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-  justify-content: left;
-  padding: 10px 12px;
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 15px;
-  line-height: 20px;
-  height: 40px;
-  border-style: none;
-  color: black;
-  background: #fff5e1;
-}
+  .button2 {
+    grid-column-start: 1;
+    grid-column-end: 3;
+    color: white;
+    background: #ff6b00;
+    border: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+  }
 
-.orange-button {
-  width: 100%;
-  height: 48px;
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 17px;
-  line-height: 24px;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  text-align: center;
-  letter-spacing: 0.03em;
-  background: #ff6b00;
-  color: white;
-  text-decoration: none;
-}
+  .button2:hover {
+    color: #f55800;
+    background: #fff5e1;
+    border: 1px solid #f55800;
+  }
 
-.orange-button:hover {
-  width: 100%;
-  height: 48px;
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 17px;
-  line-height: 24px;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  text-align: center;
-  letter-spacing: 0.03em;
-  color: #ff6b00;
-  background: #fff5e1;
-  border: 1px solid #ff6b00;
-  text-decoration: none;
-}
+  .wrapper-cards {
+    display: grid;
+    grid-template-columns: repeat(3, 31.29%);
+    grid-template-rows: repeat(2, 435px);
+    margin-left: 9.79%;
+    margin-right: 9.79%;
+    margin-top: 24px;
+    column-gap: 24px;
+    row-gap: 37px;
+  }
 
-.wrapper-cards {
-  background-color: white;
-  display: grid;
-  grid-template-columns: 100%;
-  grid-template-rows: repeat(6, 386px);
-  margin-left: 6.4%;
-  margin-right: 6.4%;
-}
+  .card {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    width: 100%;
+    height: 435px;
+    margin-top: 0%;
+  }
 
-.card {
-  display: flex;
-  flex-direction: column;
-  justify-content: flex-end;
-  width: 100%;
-  height: 386px;
-  background:
-
-    linear-gradient(
-      180.45deg,
-      rgba(38, 38, 38, 0) 0.75%,
-      rgba(38, 38, 38, 0.9) 61.94%
-    ),
-    url(./images/Placeholder2.png);
-  background-repeat: no-repeat;
-}
+  .orange-button {
+    width: 100%;
```

Comment:
- [ ] Kindly fix the styling of this button. Refer to the Figma design template.

![image](https://user-images.githubusercontent.com/92215878/206750418-efc05e94-00ee-4529-b926-1a84e8e153c7.png)